experimental/benchmarks/sdxl/conftest.py

```diff
@@ -0,0 +1,144 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--goldentime-rocm-e2e-ms",
+        action="store",
+        type=float,
+        help="Golden time to test benchmark",
+    )
```

Comment:
(This can be deferred to a follow-up refactoring, to keep this PR more incremental)

We might not need flags for these options now that the files are living in the same repo.

I'd prefer for the workflow files to be relatively frozen, with expected results stored in source files in the tests (or very close to the tests). We could pull from a source file like this: https://github.com/iree-org/iree/blob/main/build_tools/benchmarks/common/benchmark_thresholds.py

This puts _default_ values in a source file but then none of the defaults actually matter, since CI jobs always override them with CLI flags. When a developer runs the test they shouldn't _also_ need to match exactly the flags used in the workflow file.